lib/kube/proxy/forwarder.go

```diff
@@ -62,6 +62,18 @@ import (
 	utilexec "k8s.io/client-go/util/exec"
 )
 
+// KubeServiceType specifies a Teleport service type which can proxy Kubernetes requests
+type KubeServiceType int
+
+const (
+	// KubeService is a Teleport kubernetes_service
```

Comment:
I think these `KubeServiceType` values might need a bit more info about what they imply about the k8s service they apply to.

Reply:
I've expanded on these comments in the latest commit.